creator-node/src/services/sync/secondarySyncFromPrimary.js

```diff
@@ -571,11 +567,7 @@ const handleSyncFromPrimary = async ({
       // track that sync for this user was successful
       await SyncHistoryAggregator.recordSyncSuccess(fetchedWalletPublicKey)
 
-      logger.info(
-        `Sync complete for wallet: ${wallet}. Status: Success. Duration sync: ${
-          Date.now() - start
-        }. From endpoint ${creatorNodeEndpoint}.`
-      )
+      // for final log check the _secondarySyncFromPrimary function
```

Comment:
imo these comments unnecessary, but nbd

Reply:
only reason i added was because i feel like i look for best logs to search for end of sync. this just indicates where to look

Reply:
fine with me